### DISCUSSION TITLE
RuntimeWarning: invalid value encountered in double_scalars using SCS

### DISCUSSION THREAD

**Discussion by siyuanpurdue** [cvxpy/cvxpy#1393] [Q&A] [answered]
I am trying to solve a problem that involves absolute value (not sure if that's the reason causing the error),

But if I use SCS, I have this error warning: RuntimeWarning: invalid value encountered in double_scalars using SCS.

However, if I switch to MOSEK, I encounters this error.

 raise Error(rescode(res),msg)

Error: The lower bound specified is not a number (nan).

So I am not sure what's the reason behind this issue.

The following is simplified objective and constraints.
```
  obj = cp.Minimize(sum((beta[i]* mu1_vec[i] *                            \
            (abs(1+(agg_battery[3]-battery_ind[i][3])/agg_battery[3]))*      \
                agg_battery[0]- (1-agg_battery[0])*agg_battery_SOC[time-1] + \
             bld_decision[time]) for i in range(num_buildings)))              
  constraints = []
    
  for i in range(num_buildings):
        constraints += [beta[i] * 16 <= 4]
        constraints += [beta[i] * 16 <= 4]
    
   constraints += [sum(beta[i] for i in range(num_buildings)) <= 1, \
                   sum(beta[i] for i in range(num_buildings)) >= 1]
```

**Comment by SteveDiamond** [comment, marked as answer]
You definitely are passing ``nan`` in as one of your numeric values. You should check all the values of beta etc to see if they're nan.

**Comment by siyuanpurdue** [comment]
Thank you so much! It turns out my set up has a value of 0/0, which is
nan,  that is undefined or unrepresentable
